kubeflow/trainer/constants/constants.py

```diff
@@ -162,3 +162,8 @@
 
 # The Instruct Datasets class in torchtune
 TORCH_TUNE_INSTRUCT_DATASET = "torchtune.datasets.instruct_dataset"
+
+# The length of the UUID suffix for auto-generated TrainJob names.
+# Total name length = 1 (random letter) + 11 (UUID hex) = 12 characters
+# This keeps the name well under Kubernetes' 63 character limit for resource names.
+TRAINJOB_NAME_UUID_LENGTH = 11
```

Comment:
Can we move this to `kubernetes/constants.py`, since it is used only for Kubernetes backend for now.
We need to do refactoring after @szaher and @Fiona-Waters introduced local and container backends.